sockeye/data_io.py

```diff
@@ -50,8 +50,8 @@ def define_buckets(max_seq_len: int, step=10) -> List[int]:
 
 def define_parallel_buckets(max_seq_len_source: int,
                             max_seq_len_target: int,
-                            bucket_width=10,
-                            length_ratio=1.0) -> List[Tuple[int, int]]:
+                            bucket_width: int = 10,
+                            length_ratio:float = 1.0) -> List[Tuple[int, int]]:
```

Comment:
nit-picky, but one line has a space between the colon and the type and the other doesn't.